server/src/api.ts

```diff
@@ -28,36 +23,37 @@ app.use(
 );
 app.use(express.json({ limit: '1mb' }));
 
-const port = config.httpPort;
-
 const server = http.createServer(app);
 
-const wss = new WebSocket.Server({ server });
-
-interface FindPlaysQuery extends PaginationParams {
+interface FindPlaysByPlayerParams {
   player_account_hash: string;
 }
 
-async function initAPI() {
+interface FindPlaysByRoundParams {
+  round_id: string;
```

Comment:
Shouldn't it be `number`?

Reply:
it is a query param, everything is a string there, this is an interface that we use in the request params, where everything is a string